src/core/redux/selectors/selectAttachments.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+import { Map } from 'immutable';
+
+import { ATTACHMENTS } from '../../../containers/access/src/reducers/constants';
+import { ACCESS } from '../constants';
+
+export default function selectAttachments() {
+  return (state :Map) :Map => state.getIn([ACCESS, ATTACHMENTS], Map());
```

Comment:
It is possible that `.getIn` will return `undefined` if the key exists but the value is undefined, in which case the fallback is ignored. As much as that is an edge case, I think the following is safer:

```suggestion
  return (state :Map) :Map => state.getIn([ACCESS, ATTACHMENTS]) || Map();
```